hire/hire/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for hire project.
+
+Generated by 'django-admin startproject' using Django 3.1.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'whw@5v^e^ub!93z$b5)3s!ees$k*anv63z2zyjcabjec8hc4f%'
+
```

Comment:
This would normally be something to be concerned about -- SECRET_KEYs shouldn't be checked into a code repository.

However, this is the first-autogenerated version. This won't actually be a secret we will actually use as we deploy this.

Reply:
Per our discussion, let's resolve the issue before we merge.

Reply:
Understood. This has now been removed from this PR.